endpoints/events/event_test.go

```diff
@@ -677,3 +678,70 @@ func TestEventRequestToUrl(t *testing.T) {
 		})
 	}
 }
+
+func TestReadIntegrationType(t *testing.T) {
+	testCases := []struct {
+		description             string
+		givenHttpRequest        *http.Request
+		givenEventRequest       *analytics.EventRequest
+		expectedIntegrationType string
+		expectedError           error
+	}{
+		{
+			description:      "Integration type in http request is valid, expect same integration time and no errors",
+			givenHttpRequest: httptest.NewRequest("GET", "/event?t=win&b=bidId&f=b&ts=1000&x=1&a=accountId&bidder=bidder&int=TestIntegrationType", strings.NewReader("")),
+			givenEventRequest: &analytics.EventRequest{
+				Type:        analytics.Imp,
+				BidID:       "bidid",
+				AccountID:   "accountId",
+				Bidder:      "bidder",
+				Timestamp:   1234567,
+				Format:      analytics.Blank,
+				Analytics:   analytics.Enabled,
```

Comment:
You can simplify all of the `givenEventRequest` objects defined in these test cases by just defining an empty object since none of these values are relevant to the tests and `Integration` can just take on the default empty string value.